<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTodoDialog</class>
 <widget class="QDialog" name="NewTodoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>184</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="doneButton">
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>258</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="textLayout">
    <item>
     <widget class="QLabel" name="createLabel">
      <property name="text">
       <string>Create a new todo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>176</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="infoLayout">
    <item>
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Not Started</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In Progress</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Completed</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="priorityLayout">
      <item>
       <widget class="QLabel" name="priorityLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Priority:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="priorityComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="dueDateLayout">
      <item>
       <widget class="QLabel" name="dueDateLabel">
        <property name="text">
         <string>Due Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dueDateEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
